src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsEnums.cs

```diff
@@ -250,7 +251,8 @@ public enum FeatureExtension : uint
             AzureSQLSupport = 1 << (TdsEnums.FEATUREEXT_AZURESQLSUPPORT - 1),
             DataClassification = 1 << (TdsEnums.FEATUREEXT_DATACLASSIFICATION - 1),
             UTF8Support = 1 << (TdsEnums.FEATUREEXT_UTF8SUPPORT - 1),
-            SQLDNSCaching = 1 << (TdsEnums.FEATUREEXT_SQLDNSCACHING - 1)
+            SQLDNSCaching = 1 << (TdsEnums.FEATUREEXT_SQLDNSCACHING - 1),
+            JsonSupport = 1 << (TdsEnums.FEATUREEXT_JSONSUPPORT - 1)
```

Comment:
Can we name it as JSONSupport  to keep it consistent with other naming conventions such as UTF8Support?

Reply:
I would suggest not doing that. Naming conventions for C# are to capitalize the first letter of a word (ie, PascalCase), regardless of whether it is an acronym or not. There could be an argument to maintain the existing precedent, but considering there are mismatched precedents all over the codebase, I think it would be better to follow the C# convention going forward rather than maintain an incorrect precedent.